examples/kitchen-sink/webpack.config.js

```diff
@@ -1,32 +1,31 @@
 const path = require('path')
-const { getWebpackConfig } = require('@financial-times/dotcom-build-webpack-config')
-const codeSplitting = require('@financial-times/dotcom-build-code-splitting')
-const bower = require('@financial-times/dotcom-build-bower-resolve')
-const sass = require('@financial-times/dotcom-build-sass')
-const js = require('@financial-times/dotcom-build-js')
+const { PageKitBasePlugin } = require('@financial-times/dotcom-build-base')
+const { PageKitCodeSplittingPlugin } = require('@financial-times/dotcom-build-code-splitting')
+const { PageKitBowerResolvePlugin } = require('@financial-times/dotcom-build-bower-resolve')
+const { PageKitSassPlugin } = require('@financial-times/dotcom-build-sass')
+const { PageKitJsPlugin } = require('@financial-times/dotcom-build-js')
 
-module.exports = getWebpackConfig(
-  {
-    entry: {
-      scripts: './client/main.js',
-      styles: './client/main.scss',
-      async: './client/async.scss',
-      'page-kit-layout-styles': require.resolve('@financial-times/dotcom-ui-layout/styles.scss')
-    }
+module.exports = {
+  entry: {
+    scripts: './client/main.js',
+    styles: './client/main.scss',
+    async: './client/async.scss',
+    'page-kit-layout-styles': require.resolve('@financial-times/dotcom-ui-layout/styles.scss'),
   },
-  [
-    codeSplitting.plugin(),
-    bower.plugin(),
-    sass.plugin({
+  plugins: [
+    new PageKitBasePlugin(),
+    new PageKitCodeSplittingPlugin(),
+    new PageKitBowerResolvePlugin(),
+    new PageKitSassPlugin({
       // Enabling webpackImporter because Sass itself can only resolve partial files based on the
       // CWD and not relative to the current file being processed. This means Sass can't find the
       // nested dependencies created when symlinking.
       webpackImporter: true,
       includePaths: [
         // All Bower components are installed at the repo root
-        path.resolve('../../bower_components')
-      ]
+        path.resolve('../../bower_components'),
+      ],
     }),
-    js.plugin()
-  ]
-)
+    new PageKitJsPlugin(),
```

Comment:
Super minor, but could we move the JS plugin above the sass one? It might be easier to see that it's part of the same array, with sass being the only one to have extra config options.